lib/utils.js

```diff
@@ -63,14 +63,16 @@ const clone = exports.clone = function clone(obj, options) {
  */
 
 exports.cloneObject = function cloneObject(obj, options) {
-  const minimize = options && options.minimize;
-  const ret = {};
-  let hasKeys;
-  let val;
-  const keys = Object.keys(obj);
-  let k;
-
-  for (let i = 0; i < keys.length; ++i) {
+  const minimize = options && options.minimize,
```

Comment:
Minor nitpick, but we're trying to move away from `,` declarations in favor of one declaration per line. So I think this change is a step in the wrong direction from a purely stylistic perspective. But not a blocker here.